.gitignore

```diff
@@ -31,3 +31,4 @@ webviews-metafile.json
 .eslintcache
 .nyc_output
 samples/connection-sharing-sample/dist
+dist
```

Comment:
out contains tsc compiled code for tests where as dist contains actual bundled code that runs.